src/Metal/MTLEnums.cs

```diff
@@ -191,6 +195,29 @@ public enum MTLVertexFormat : nuint {
 		
 		Int1010102Normalized = 40,
 		UInt1010102Normalized = 41,
+
+		[Mac (10,13), iOS (11,0), TV (11,0), NoWatch]
+		UChar4Normalized_Bgra = 42,
```

Comment:
This one has the underscore to match the obj-c version: ```MTLVertexFormatUChar4Normalized_BGRA```.  I can remove it though if that's the preference